app/api/organization/controller.js

```diff
@@ -146,13 +146,13 @@ exports.createOrganizationCase = async (req, res) => {
 exports.deleteOrganizationCase = async (req, res) => {
   const {
     user: { organization_id },
-    body: { case_id }
+    body: { caseId }
```

Comment:
Change this from snake to camelCase to keep consistent with formatting on other request body params.